### ISSUE TITLE
Known Devices disappear in latest Release (2.31.25.20180324 on RPI3)

### ISSUE THREAD

**Issue by khillman** [jens-maus/RaspberryMatic#307]
Hi, 

i'm not sure whether this is only a problem of my local raspmatic instance (running on a rpi3) or a problem of all such instances in that version...

I've got currently the newest Version (https://github.com/jens-maus/RaspberryMatic/releases/download/2.31.25.20180324/RaspberryMatic-2.31.25.20180324-rpi3.zip) running (coming from an older Version from the last year which was working stable but does not support a new HmIP outdoor temperature and humidity sensor) and it seems that it looses devices from time to time... actually at least two PSM HmIP were lost the third time... everytime i have to reset them to factory default and relearn them to the raspmatic. At the beginning after the update from the old version i already had some problems with a couple of HmIP TRV2/SWDO devices. Additionally the raspmatic sometimes "freezes" on "loading" when i try to access the devices state overview using the webinterface.

So there are multiple questions: 
- Are there other instances using the newest version which have same/similar problems?
- Is the problem already known / are there any ideas what can cause this behaviour?
- Is there an upcoming release which fixes that or should i go back to an older version?

**Comment by Chris5917** [comment]
Hi khillman,

same thoughts here. I am not using RaspberryMatic, I just transitioned from YAHM to piVCCU. I used the opportunity to reset and register all my devices and configure my homematic environment from scratch. I thought I am crazy or did somehting wrong, but devices just configured to the last missing setting disappeared again once they left the inbox for registering. I'm down to the detail that it happened during a reboot of the CCU, after the  next start they just did not exist anymore and I had to re-register them. I have around 50 devices (no hmIP) and it happened around 2-3 times to different devices. 

As we have the same perception while using different solution approaches, the root cause maybe more likely in a commonly used component, maybe it's FW 2.31.25 itself (in Addition I have the email addon, CUxD latest Version, and quickaccess). I don't have the technical skills to investigate this, just my experiences and perceptions.

BR
  Chris
